comb_spec_searcher/strategies/constructor.py

```diff
@@ -319,8 +329,8 @@ def get_extra_parameters(
         child match the parents parameters.
         """
         res: List[Optional[Dict[str, int]]] = []
-        for extra_parameters in self.extra_parameters:
-            update_params: Dict[str, int] = {}
+        for i, extra_parameters in enumerate(self.extra_parameters):
+            update_params: Dict[str, int] = {**self.fixed_values[i]}
```

Comment:
Is this just another way of doing
`update_params = dict(self.fixed_values[i])`
?

Reply:
yeah, its the same thing, I could have done copy here for example